meta-balena-common/recipes-connectivity/bluez5/bluez5_%.bbappend

```diff
@@ -14,3 +14,5 @@ do_install_append() {
 
     install -d ${D}/var/lib/bluetooth
 }
+
+EXTRA_OECONF = "--enable-sixaxis "
```

Comment:
@zvin. Nice work. One tiny fix. That should be `+=` instead of `=`. Otherwise we override anything we inherit from upstream.

With `+=`, we will append `--enable-sixaxis` and keep anything previously defined in other layers.

Also, the whitespace at the end isn't needed with `+=`. That is needed with `_prepend` or `_append`. Yocto can be confusing and I mix those up too all the time..

Reply:
Updated, thanks @ZubairLK